pronotepy/dataClasses.py

```diff
@@ -376,7 +379,8 @@ class Lesson:
         'estRetenue': ('detention', bool, False),
         'estSortiePedagogique': ('outing', bool, False),
         'dispenseEleve': ('exempted', bool, False),
-        'listeVisios,V': ('virtual_classrooms', lambda l: [i["url"] for i in l], [])
+        'listeVisios,V': ('virtual_classrooms', lambda l: [i["url"] for i in l], []),
+        'P': ('num', str)
```

Comment:
Possibly change to int? Does it have to be a string?

Reply:
Yes, messed up here.
I changed it.